README.md

```diff
@@ -170,10 +170,10 @@ If you don't know the path to the Android SDK, you can find it using
 Android Studio by going to the SDK Manager tool.
```

Comment:
FYI there's now also a script at `tools/install-android-sdk` to install the android sdk. You still have to do the PATH stuff yourself though (it spits out the line you need to add).

Not sure if that's useful, but I thought I'd mention it